matsim/src/main/java/org/matsim/api/core/v01/population/Plan.java

```diff
@@ -49,6 +49,18 @@ public interface Plan extends MatsimPopulationObject, Customizable, BasicPlan, A
 	public abstract String getType();
 
 	public abstract void setType(final String type);
+	
+	public abstract void setPlanId(final String planId);
+	
+	public abstract String getPlanId();
```

Comment:
Thanks for this great work! Maybe it would make sense to let `Plan` implement the [`Identifiable`](https://github.com/matsim-org/matsim-libs/blob/25fefcd4b04ba6909d2db583c4cbe267aa5fbc2f/matsim/src/main/java/org/matsim/api/core/v01/Identifiable.java#L23) interface and use the regular MATSim Id class for ids. This would allow using the specialized container classes `IdMap`, which are very efficient.

Reply:
This should be implemented now.

Reply:
Looks great!